packages/gasket-cli/lib/bin/boot.js

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env node
-
 const { join } = require('path');
 const location = ['@gasket', 'cli', 'bin', 'run'];
```

Comment:
Should this now look in lib?
```suggestion
const location = ['@gasket', 'cli', 'lib', 'bin', 'run'];
```